doc/conf.py

```diff
@@ -80,24 +83,30 @@
 # ==============================================================================
 # Options for HTML output
 # ==============================================================================
-html_theme_options = {
-    'home_breadcrumbs': True,
-    'vcs_pageview_mode': 'blob',
-}
-
 html_context = {}
-ctx = Path(__file__).resolve().parent / 'context.json'
+ctx = ROOT / 'context.json'
 if ctx.is_file():
 	html_context.update(loads(ctx.open('r').read()))
 
-html_theme_path = ["."]
-html_theme = "_theme"
+if (ROOT / "_theme").is_dir():
+	html_theme_path = ["."]
+	html_theme = "_theme"
+	html_theme_options = {
+		'logo_only': True,
+		'home_breadcrumbs': False,
+		'vcs_pageview_mode': 'blob',
+	}
+else:
+	html_theme = "alabaster"
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
+html_logo = str(Path(html_static_path[0]) / "icon.png")
```

Comment:
I think the logo is to big in the navigation bar.

![image](https://user-images.githubusercontent.com/956109/149711747-d717c63d-69a3-4143-9121-8eea30984d94.png)

Reply:
Updated to:

![image](https://user-images.githubusercontent.com/38422348/155405422-ff6b985e-5940-458f-a62c-9c74a648b93e.png)